numpy/core/tests/test_arrayprint.py

```diff
@@ -796,6 +797,46 @@ def test_dtype_linewidth_wrapping(self):
             array(['1', '1', '1', '1', '1', '1', '1', '1', '1', '1', '1', '1'],
                   dtype='{}')""".format(styp)))
 
+    @pytest.mark.parametrize(
```

Comment:
Won't this test fail on big-endian systems?

Reply:
Yep, good point.
I think what I'll do is
1. rename `little_end` to `native` and `big_end` to `non_native`
```py
import sys
if sys.byteorder == 'big':
    # flip native and non-native for big-endian systems
    non_native = non_native.replace('>', '<')
    native = native.replace('<', '>')
```